Task 2/Task 2.1/Task 2.1/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Task_2._1
+{
+    class Program
+    {
+        static List<Figure> listFig = new List<Figure>();
+        static void Main(string[] args)
+        {
+            do
+            {
+                int answer = ConsoleHelper.IntReadParse("\nВыбирете действие\n" +
+                    "1.Добавить фигуру\n" +
+                    "2.Вывести фигуры\n" +
+                    "3.Очистить холст\n" +
+                    "4.Выход", 1, 4);
+                if (answer == 4) break;
+                switch (answer)
+                {
+                    case 1:
+                        AddFigure();
+                        break;
+                    case 2:
+                        PritFig();
+                        break;
+                    case 3:
+                        listFig.Clear();
+                        break;
+                    default:
+
+                        break;
+                }
+
+            } while (true);
+
+
+        }
+
+        public static void PritFig()
+        {
+            foreach (var item in listFig)
+            {
+                Console.WriteLine(item.Name);
+            }
+        }
+        public static void AddFigure()
+        {
+            Console.WriteLine(@"
+        Circle = 1,
+        Ring=2,
+        Rectangle=3,
+        Square=4");
+
+                        int tip = ConsoleHelper.IntReadParse("Введите тип фигуры", 1, 4);
+
+            int x, y, r, r2;
+            switch (tip)
+            {
+                case (int)TypeFigure.Circle:
+                    x = ConsoleHelper.IntReadParse("Введите x кориднату центра", 0, int.MaxValue);
+                    y = ConsoleHelper.IntReadParse("Введите y кориднату центра", 0, int.MaxValue);
+                    r = ConsoleHelper.IntReadParse("Введите radius", 1, int.MaxValue);
+                    listFig.Add(new Circle(new Position(x, y), r));
+                    break;
+
+                case (int)TypeFigure.Ring:
+                    x = ConsoleHelper.IntReadParse("Введите x кориднату центра", 0, int.MaxValue);
+                    y = ConsoleHelper.IntReadParse("Введите y кориднату центра", 0, int.MaxValue);
+                    r = ConsoleHelper.IntReadParse("Введите radius внутрений", 1, int.MaxValue);
+                    r2 = ConsoleHelper.IntReadParse("Введите radius внешний", 1, int.MaxValue);
+                    listFig.Add(new Ring(new Position(x, y), r, r2));
+                    break;
```

Comment:
It's good that program doesn't let me to create a ring with inner radius bigger than outer, but it's bad that exception isn't catched and the whole app just crashes. try/catch can help with that.